src/main/java/frc/robot/Robot.java

```diff
@@ -36,6 +45,13 @@ public void robotInit() {
     DriveBase.init();
 
     m_robotContainer = new RobotContainer();
+    
+    //create a vision control table
```

Comment:
It is nice that you are documenting your code but for simple straightforward concepts they don't give any extra value to the reader so it may be worth removing theses ones.